webpack.config.js

```diff
@@ -22,7 +22,22 @@ const CHARTS = {
     tooltip: `${PATHS.charts}/tooltip/tooltipComponent.js`,
 };
 
+const BABEL_LOADER_MODULE = {
+    rules: [
+        {
+            test: /\.js$/,
```

Comment:
Are we not planning to use *.jsx for React components?

Reply:
Usually we don't, but I don't know why. Maybe @BenAtEventbrite  could point out the reason.

Reply:
The industry has moved away from `*.jsx` extensions. When React first came out it was commonplace, but then it fell out of favor. I'm not 100% sure why, but my guess is because React is "just JavaScript". So it shouldn't be any different than other files.

At EB specifically, we already had all of this tooling that assumed `*.js` only for JavaScript, so it would've been unnecessarily painful to support `*.jsx` across the entire infrastructure.

Reply:
Cool, thanks for the explanation @benmvp !